test/libsolidity/syntaxTests/pragma/invalid_range_operator_range_first.sol

```diff
@@ -0,0 +1,3 @@
+pragma solidity <0.8.17 - 0.8.20;
```

Comment:
There are not that many operators to test (`>`, `>=`, `<`, `<=`, `=`, `^`, `~`) so it would be good to test them all, at both positions.

You can check multiple cases at the same time by using a multi-file test. E.g.:
```
==== Source: A ====
pragma solidity <0.8.17 - 0.8.20;
==== Source: B ====
pragma solidity <=0.8.17 - 0.8.20;
==== Source: C ====
pragma solidity 0.8.17 - >0.8.20;
```